cosmosdb/types.bal

```diff
@@ -299,20 +302,21 @@ public type DocumentCreateOptions record {|
     boolean isUpsertRequest = false;
 |};
 
-# Represent the optional parameters which can be passed to the function when replacing a document.
+# Optional parameters which can be passed to the function when replacing a document.
 # 
-# + indexingDirective - The option whether to include the document in the index. Allowed values are 'Include' or 
-#                       'Exclude'.
+# + indexingDirective - The option whether to include the document in the index
+#                     - Allowed values are 'Include' or 'Exclude'.
 @display{label: "Document Replace Options"}
 public type DocumentReplaceOptions record {|
     @display{label: "Indexing Option"}
     IndexingDirective indexingDirective?;
 |};
 
-# Represent the optional parameters which can be passed to the function when listing information about the documents.
+# Optional parameters which can be passed to the function when listing information about the documents.
 # 
-# + consistancyLevel - The consistency level override. Allowed values are `Strong`, `Bounded`, `Session` or `Eventual`.
-#                      The override must be the same or weaker than the Cosmos DB account’s configured consistency level.
+# + consistancyLevel - The consistency level override
```

Comment:
Suggestion: "The consistency level override. Possible values are `Strong`, `Bounded`, `Session`, and `Eventual`. The override must be the same or weaker than the Cosmos DB account’s configured consistency level."